web/api/formatters/json_wrapper.c

```diff
@@ -26,16 +26,29 @@ void rrdr_json_wrapper_begin(RRDR *r, BUFFER *wb, uint32_t format, RRDR_OPTIONS
                        "   %sapi%s: 1,\n"
                        "   %sid%s: %s%s%s,\n"
                        "   %sname%s: %s%s%s,\n"
-                       "   %sview_update_every%s: %d,\n"
+                   , kq, kq
+                   , kq, kq, sq, r->st->id, sq
+                   , kq, kq, sq, r->st->name, sq);
+
+    RRDHOST *host = r->st->rrdhost;
+    buffer_sprintf(wb, "   %slabels%s: {\n", kq, kq);
+    int count = 0;
```

Comment:
A suggestion, I verified that the variable `count` is incremented during the loop and its main goal is to avoid to print a break line when we are in the first address of the loop, I think you can discard this variable and save the first address `host->labels`, with this you can compare the address values and print the break line only when the addresses are different.